### PR TITLE
MKT-1892 -- Logged in Nav More like Product

### PR DESCRIPTION
**Pull request by stewsmith**
1. Change "Dashboard" to "Home"
2. Remove the experiments dropdown
3. Add link to /appsettings under account

https://optimizely.atlassian.net/browse/MKT-1892

### CODE REVIEW

File: website-guts/templates/client/loggedInUtilityNav.hbs [hunk 1]
```diff
@@ -1,47 +1,5 @@
...
-          <span>View All Experiments</span>
-        </a>
-      </li>
-      {{#if showCreateLink}}
-      <li class="show-create-experiment hide-in-mobile" data-modal-click="create-exp">
-        <a>Create New Experiment</a>
-      </li>
-      {{/if}}
-    </ul>
+  <a href="{{apiDomain}}/dashboard?project_id={{account_id}}">Home</a>
```
Comment (jeffz-optimizely) [inline website-guts/templates/client/loggedInUtilityNav.hbs]: This should actually redirect to `/projects/`, maybe with the account id (i.e. `/projects/{{account_id}}`. @jordangarcia Can you advise what the proper link is in this case?

Comment (jeffz-optimizely) [inline website-guts/templates/client/loggedInUtilityNav.hbs]: From looking at the nav item in product, it links to `/projects/{{account_id}}/experiments`, so we should just match that.

Comment (jordangarcia) [inline website-guts/templates/client/loggedInUtilityNav.hbs]: `/projects/` will give the admin account project or the first non-archived project.

**Comment by jeffz-optimizely** [comment]
QA'd in person with Stew. LGTM!

File: website-guts/templates/client/loggedInUtilityNav.hbs [hunk 2]
```diff
@@ -53,6 +11,9 @@
       <li class="hide-in-mobile">
```
Comment (dtothefp) [inline website-guts/templates/client/loggedInUtilityNav.hbs]: @stewsmith this is unrelated but can you add the `apiDomain` to the mobile email for `/account`

```
<a id="mobile-email-nav" class="customer-email" href="/account">{{email.mobile}}</a>
```

**Comment by dtothefp** [comment]
LGTM

**Pull request merged**